.github/workflows/publish-release-assets.yml

```
@@ -0,0 +1,214 @@
+  schedule:
+    - cron: "0 2 * * *" # Every day at 2:00am
+  release:
+    types: [published]
+
+jobs:
+  check-version:
+    name: Check the version validity
+    runs-on: ubuntu-latest
+    # No need to check the version for dry run (cron)
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)